testing/MLDB-2143-classifier-utf8.py

```diff
@@ -63,13 +63,14 @@ def test_utf8_category(self):
         })
 
         runResults = rez.json()["status"]["firstRun"]["status"]["folds"][0]["resultsTest"]["weightedStatistics"]
+
         self.assertEqual(runResults, {
-        "recall": 1.0,
-        "support": 72.0,
-        "f1Score": 1.0,
-        "precision": 1.0,
-        "accuracy": 1.0
-    })
+            "recall": 0.9444444444444444,
+            "support": 72.0,
+            "f1Score": 0.9446548821548821,
+            "precision": 0.9537037037037037,
+            "accuracy": 0.9645061728395061
+        })
```

Comment:
Why are those values changing? Were they incorrect before? Or because the only feature that was used was the label?